frontend/plugins/insurance_ui/src/pages/insurance/ContractTemplatesPage.tsx

```
@@ -66,7 +68,7 @@ export default function ContractTemplatesPage() {
   };
 
   const handleDelete = async (id: string) => {
-    if (confirm('Delete this template?')) {
+    if (confirm(t('delete-template-confirm'))) {
```

ESLint

[error] Unexpected use of 'confirm'.

(no-restricted-globals)